cdap-ui/app/hydrator/controllers/detail/canvas-ctrl.js

```diff
@@ -68,26 +68,29 @@ angular.module(PKG.name + '.feature.hydrator')
               rDisabled: function() {
                 return true;
               },
-              rNodeMetricsContext: function() {
-                var appParams = HydratorPlusPlusDetailRunsStore.getParams();
-                var latestRun = HydratorPlusPlusDetailRunsStore.getLatestRun();
-                var logsParams = HydratorPlusPlusDetailRunsStore.getLogsParams();
-                let runs = HydratorPlusPlusDetailRunsStore.getRuns();
+              rNodeMetricsContext: ['$stateParams', 'GLOBALS', function($stateParams, GLOBALS) {
+                let pipelineDetailStoreState = window.CaskCommon.PipelineDetailStore.getState();
```

Comment:
can you try removing these array type DI, and use `ngInject`? Do a `gulp distribute` afterwards to test. I think ngInject should be able to do it now.